torch/_inductor/fx_passes/group_batch_fusion.py

```diff
@@ -396,10 +396,14 @@ def _pointwise_node_can_be_fused(self, node: torch.fx.Node):
         input, other = node.args
         return (
             input.meta["val"].shape == other.meta["val"].shape  # type: ignore[union-attr]
+            # input and other can be scalars, where they have no attribute 'meta'
             if hasattr(input, "meta")
             and hasattr(other, "meta")
-            and "val" in input.meta  # type: ignore[union-attr]
-            and "val" in other.meta  # type: ignore[union-attr]
+            and is_node_meta_valid(input)  # type: ignore[arg-type, union-attr]
+            and is_node_meta_valid(other)  # type: ignore[arg-type, union-attr]
+            # torch.SymInt or torch.SymFloat object has no attribute 'shape'
+            and hasattr(input.meta["val"], "shape")  # type: ignore[union-attr]
+            and hasattr(other.meta["val"], "shape")  # type: ignore[union-attr]
```

Comment:
Wouldn't it be better to test if val is a torch.Tensor?

Reply:
Sure, I can check if it's FakeTensor